turbinia/lib/libcloudforensics.py

```diff
@@ -437,6 +437,66 @@ def get_or_create_analysis_vm(
     created = True
     return instance, created
 
+  def list_instance_by_label(self, labels_filter=''):
+    """Lists VMs in a project with on of the provided labels.
+
+    Args:
+      labels_filter: a dict of labels to find --> {'id': '123'}
+
+    Returns:
+      A call to __list_by_label with the proper instance_service_object
```

Comment:
It would be better here to have the actual return type of `__list_by_label`

Reply:
Done